models/database.go

```diff
@@ -550,13 +550,9 @@ var databaseSchema = [][]string{
 		ADD COLUMN agent_password VARCHAR CHECK (agent_password <> '')`,
 	},
 	43: {
-		`CREATE TYPE software_version AS (
-  			name VARCHAR,
-  			version VARCHAR
-		);`,
 		`CREATE TABLE service_software_versions (
 			service_id VARCHAR NOT NULL CHECK (service_id <> ''),
-			software_versions software_version ARRAY,
+			software_versions JSONB,
```

Comment:
it doesn't work after all?

Reply:
inserts with empty array worked, but if it has some entries it require to define Scan() and Value() functions for the array element type, I decided to use JSONB, because it's easier to use.. plus, we use it in the other places.